### ISSUE TITLE
NVDA sonic integration

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1206]
**Reported by aleksey_s on 2010-11-18 09:58**
Scope of this ticket is integration of sonic library by Bill Cox (http://www.freelists.org/post/programmingblind/New-speech-speedup-library-available-under-GPL) into NVDA. User should be able to speed up every synth that supports autio generation  and retrieval.

**Comment by leonardder** [comment]
Sonic is now a separate dependency, although only used for espeak.

**Comment by derekriemer** [comment]
Couldn't we in theory use it for everything?

**Comment by Adriani90** [comment]
cc: @leonardder comming back on this since there were actually many issues where users requested similar behaviors between synths like higher speed for narator etc. This is maybe also interesting to consider.

**Comment by Adriani90** [comment]
The sonic library is integrated as dependency in NVDA as Leonard pointed above already. Thus this issue is fixed. I suggest to open new issues for specifically synths which would benefit from usage of Sonic. Closing.
